cli/bptest/cmd.go

```diff
@@ -0,0 +1,51 @@
+package bptest
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var flags struct {
+	testDir string
+}
+
+func init() {
+	viper.AutomaticEnv()
+	Cmd.AddCommand(listCmd)
+
+	listCmd.Flags().StringVar(&flags.testDir, "test-dir", "test/integration", "Path to directory containing integration tests")
```

Comment:
Were you thinking recursive look up the tree since users commonly edit files within the repo dirs? I thought about it but deferred since commonly folks would `make docker_*` and land in the right place and `test-dir` is available for other scenarios.

I can add this as a follow up after we complete the other core functionality?

Reply:
More like recursively looking down from the current working directory. In particular, is there a reason we need to only look in `test/integration` for tests? Couldn't we just explore from the repo root downwards to find any tests?

Reply:
> In particular, is there a reason we need to only look in test/integration for tests? Couldn't we just explore from the repo root downwards to find any tests?

Some of my initial thoughts when implementing this were:

- For computing the [auto discovered tests](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/c65e9f897cae1b18aece87ce92d1370913871db7/infra/blueprint-test/pkg/discovery/discover.go#L54-L55), we need to find the `test/integration` dir. This is because the test discovery attempts to locates configs in `examples`/`fixtures` by traversing up the tree from `test/integration`.
- We could find `test/integration` dir by recursively looking down, however it would only find `test/integration` if the user in currently in the `repo-root` or in `repo-root/test`.

Reply:
Instead of hardcoding `test/integration`, couldn't we look for `discover_test.go` and assume examples are relative to that? This would you to execute the commands from the root, `test`, or `test/integration`?

Reply:
That should work. Today some blueprints may not have a `discover_test.go` if they are only using explicit tests, should we require all blueprints to have a `discover_test.go`? It should be no-op if no test fn is defined.

Reply:
We shouldn't need to require `discover_test.go` for blueprints with only explicit tests. It looks like the integration test directory is only used as a prefix for [the glob](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/1027/files#diff-c52c0ded7de89e9c4892c96231c7f6e286a61908ffac418418c5ee4bc7d400caR45). Is there a reason we couldn't just glob from the current (root) directory without the prefix?

Reply:
I was thinking of some modules where we may have [function code](https://github.com/terraform-google-modules/terraform-google-scheduled-function/tree/master/modules/project_cleanup/function_source) with potential tests which may have unrelated matches for `_test.go`. 

I think thats a small subset, so should be okay to proceed with this.

Reply:
Yeah and in such cases it should be sufficient to simply run the CLI from `test/` instead of the root directory.

Reply:
done